images/Dockerfile.el6

```diff
@@ -3,5 +3,9 @@ FROM centos:6
 RUN yum install -y wget epel-release && yum clean all
 RUN yum install -y https://yum.theforeman.org/client/1.21/el6/x86_64/foreman-client-release.rpm && yum clean all
 RUN yum install make gofer python-gofer-proton python-pulp-agent-lib subscription-manager python-pip -y && yum clean all
+
+# latest mock not 2.6 compatible
+RUN wget -O /usr/lib/python2.6/site-packages/mock.py https://raw.githubusercontent.com/testing-cabal/mock/1.0.1/mock.py --no-check-certificate
```

Comment:
I think you could use `RUN yum install -y python-mock` as well. Or `RUN pip install mock==1.0.1`

Reply:
That is better ... updated :)